onnx/numpy_helper.py

```diff
@@ -14,10 +15,14 @@ def combine_pairs_to_complex(fa: Sequence[int]) -> List[complex]:
 
 
 def bfloat16_to_float32(
-    data: np.ndarray, dims: Union[int, Sequence[int]]
+    data: np.ndarray, dims: Optional[Union[int, Sequence[int]]] = None
 ) -> np.ndarray:
     """Converts ndarray of bf16 (as uint32) to f32 (as uint32)."""
     shift = lambda x: x << 16  # noqa: E731
+    if dims is None:
+        if len(data.shape) == 0:
+            return shift(np.array([data]).astype(np.int32)).view(np.float32)[0]
```

Comment:
This is a bit confusing. Is `data` not an `ndarray` in this case (as the type annotation above indicates)? Is it meant to be a scalar? (I notice that the scalar-transformation function has disappeared from the helper.py.) Clarify in the type-annotation or comments as appropriate.

Reply:
I removed the version I added in helper.py to handle scalar. It seemed confusing to have two versions. I extended this one. I fixed the annotation.